src/main.cpp

```diff
@@ -1500,7 +1500,7 @@ void askForInternetPermission()
     new MessageDialog(_("SuperTuxKart may connect to a server "
         "to download add-ons and notify you of updates. We also collect "
         "anonymous hardware statistics to help with the development of STK. "
-        "Please read our privacy policy at http://privacy.supertuxkart.net. "
+        "Please read our privacy policy at https://privacy.supertuxkart.net. "
```

Comment:
This should be https://supertuxkart.net/Privacy, as privacy.supertuxkart.net does not support HTTPS.